src/test/java/org/apache/commons/beanutils2/BeanificationTestCase.java

```diff
@@ -327,9 +329,21 @@ public String toString() {
         thread.join();
 
         assertEquals("Signal not set by test thread", 2, signal.getSignal());
-        assertTrue("Different BeanUtilsBean instances per context classloader", BeanUtilsBean.getInstance() != signal.getBean());
-        assertTrue("Different ConvertUtilsBean instances per context classloader", ConvertUtilsBean.getInstance() != signal.getConvertUtils());
-        assertTrue("Different PropertyUtilsBean instances per context classloader", PropertyUtilsBean.getInstance() != signal.getPropertyUtils());
+        assertNotEquals(

```

Comment:
What's the change here? If it's only formatting, please revert, it just makes PRs noisier.

Reply:
Before: Used `assertTrue` with a condition to it.  
After: Uses `assertNotEquals` with the 2 variables passed to it.

Typically, with JUnit `assertEquals` or `assertNotEquals` is used rather than `==` or `!=`.

Happy to turn it back to a single line though, but the primary change was the assertion method that was used.